### PR TITLE
[chore] initial readme

### PR DESCRIPTION
**Pull request by garguelles**
# Changes

- basic readme

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# tesr.finance 
+
+## Initial setup
+
+- Fork the original [repository](https://test.com) in to your github account
```
Comment (pajicf) [inline readme.md:5]: I guess this should be https://github.com/MVPWorkshop/tesseract-app instead. Regarding the comment above, we can remove this step and use this link in `git clone`

**Review by pajicf — CHANGES_REQUESTED** [review]
Nice and short, great job!
Left a few comments, do you think we can also provide a more detailed .env.example as part of this PR?

**Comment by garguelles** [comment]
> Nice and short, great job! Left a few comments, do you think we can also provide a more detailed .env.example as part of this PR?

Yes, will put the values we use for local dev here

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  ```
+- Create a `.env` file in the root directory
+  ```
+  cp .env.example .env
+  ```
+
+  These are the variables we use for local development
+
+  ```
+  REACT_APP_API_BASE_URL=https://prom.tesr.finance/api/v1
```
Comment (pajicf) [inline readme.md:17]: We can use the actual .env.example file, makes cloning the project easier and easier to maintain than to have to update both the README and the .env.example file.

If we ever have values which shouldn't be "public", we just leave that place blank and add it specifically to the README on how to obtain

Comment (garguelles) [reply readme.md:17]: got it. Will add these in the example instead.

**Pull request merged**